fdbclient/Knobs.cpp

```diff
@@ -209,4 +210,8 @@ ClientKnobs::ClientKnobs(bool randomize) {
 	//fdbcli		
 	init( CLI_CONNECT_PARALLELISM,                  400 );
 	init( CLI_CONNECT_TIMEOUT,                     10.0 );
+
+	// trace
+	init( TRACE_LOG_FILE_IDENTIFIER_MAX_LENGTH,    1024);
```

Comment:
My gut says to limit this much smaller, like say 50. If you feel strongly about a longer length, though, I'm open to being persuaded otherwise.

```suggestion
	init( TRACE_LOG_FILE_IDENTIFIER_MAX_LENGTH,      50 );
```